scripts/ci_policy_checks.rb

```
@@ -33,8 +33,135 @@ def changed_files(base)
+  end
+
+  return nil if delimiter_end >= content.length || content[delimiter_end] != '('
+
+  delimiter = content[delimiter_start...delimiter_end]
+  terminator = ')' + delimiter + '"'
+  literal_end = content.index(terminator, delimiter_end + 1)
+  return nil if literal_end.nil?
+
+  literal_end += terminator.length
+  [content[start_idx...literal_end], literal_end]
+end
```

RuboCop (1.86.0)

[convention] Assignment Branch Condition size for `read_raw_string_literal` is too high. [<9, 18, 14> 24.52/23]

(Metrics/AbcSize)